templates/blink/intent_to_implement.html

```diff
@@ -132,24 +127,47 @@ <h4>Contact emails</h4>
 
 {% if 'experiment' in sections_to_show %}
 
-  <br><br><h4>Goals for experimentation</h4>
-  <p class="preformatted">{{feature.experiment_goals|urlize}}</p>
+  <br><br><h4>Origin trial info</h4>
+  <div style="margin-left: 4em;">
 
-  {% if feature.experiment_timeline %}
-    <br><br><h4>Experimental timeline</h4>
-    <p class="preformatted">{{feature.experiment_timeline|urlize}}</p>
-  {% endif %}
+    <br><br><h4>Goals for experimentation</h4>
+    <p class="preformatted">{{feature.experiment_goals|urlize}}</p>
 
-  {% if 'extension_reason' in sections_to_show %}
-    <br><br><h4>Reason this experiment is being extended</h4>
-    <p class="preformatted">{{feature.experiment_extension_reason|urlize}}</p>
-  {% endif %}
+    <br><br><h4>OT desktop start</h4>
+    {{feature.ot_milestone_desktop_start}}
+    <br><br><h4>OT desktop end</h4>
+    {{feature.ot_milestone_desktop_end}}
```

Comment:
We've seen confusion a number of times about the meaning of the "end" milestone. For example, is the end milestone inclusive (last milestone where the trial still works) or exclusive (trial ends when this milestone reaches stable).

It might be helpful to be more explicit about whether the end milestone is inclusive or exclusive.

Reply:
Historically, this has always been inclusive. A plan down the road is to show stages graphically. An inclusive end would mean there is no overlap between OT and enabled by dcefault.